compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -1419,7 +1419,73 @@ OMR::ResolvedMethodSymbol::getNumberOfBackEdges()
    return numBackEdges;
    }
 
+/*
+ * Returns the bytecode info for an OSR point, taking into account the
+ * possibility that the treetop for a call has different BCI to the OSR
+ * point.
+ */
+TR_ByteCodeInfo
+OMR::ResolvedMethodSymbol::getOSRByteCodeInfo(TR::Node *node)
```

Comment:
Should this return a reference since the nodes return a reference to save compile-time and memory?